whynotearth/BrowTricks
# Fix: Miscellaneous Cleanup

# Description

This PR does not address a particular issue, but rather fixes some small issues here and there that I felt did not warrant their own PR.  If you are a reviewer and you feel differently, please let me know and we can discuss it.

In this PR we make changes mostly involving code organization:

* d46e2db: Delete stock files that came from `vue create`, such as 'Logo.png' and 'HelloWorld.vue'
* e8f8660: Update the 'Button' component to use slots for the left-aligned icon and centered CTA text
* 831e5dc: Update the 'Text' and 'TextArea' components to use slots for their respective labels
* 25f5286: Update the current customer views to follow the new slotting patterns from the last two commits, and uninstantiate 'MenuDrawer' as it is instantiated in  'MenuHeader' now
* 2271bbc: Update 'MenuHeader' to be sticky instead of fixed

# How to test

There are no visual nor functional changes in this PR. Visit the three views and observe that there are no visual nor functional changes to speak of.

1. [Customer View - Home](https://deploy-preview-46--browtricks.netlify.app/)
2. [Customer View - Business Hours](https://deploy-preview-46--browtricks.netlify.app/shop/test-tenant-slug/business-hours)
3. [Customer View - Contact Form](https://deploy-preview-46--browtricks.netlify.app/shop/test-tenant-slug/contact)

(Links will take you to the pages in the deploy preview)
